commercetools-java-client-core/src/main/java/io/sphere/sdk/client/TestDoubleSphereClientFactory.java

```diff
@@ -55,7 +55,9 @@ public String toString() {
 
             @Override
             public SphereApiConfig getConfig() {
-                return null;
+
+                return SphereApiConfig.of("DUMMY_PROJECT_KEY","DUMMY_API_URL");
```

Comment:
Notice there's another `getConfig` some lines below this code.
Also notice that this client's `execute` method already uses a `SphereApiConfig`. Why don't we better save the config as a class field and use it in both sides?